numpy/random/src/distributions/distributions.h

```diff
@@ -9,7 +9,7 @@
 #include "Python.h"
 #include "numpy/npy_common.h"
 #include "numpy/npy_math.h"
-#include "numpy/random/bitgen.h"
+#include "src/bitgen.h"
```

Comment:
This isn't correct for 1.17, and it looks like 1.17 will be the same as 1.18 after the C-API changes go in.
```
charris@fc [numpy.git ((v1.17.4))]$ find . -name 'bitgen\.h'
./numpy/core/include/numpy/random/bitgen.h
```

Reply:
The current 1.17 file is renamed, do we want to do that for an existing release?